yogstation/code/game/objects/items/fishing/rods.dm

```diff
@@ -0,0 +1,149 @@
+/obj/item/twohanded/fishingrod
+	name = "fishing rod"
+	desc = "A fishing rod used for fishing. Despite looking ordinary, fishing has evolved for the cosmos with features like auto-reeling."
```

Comment:
Suggestion: "A rod used for fishing. Despite ordinary appearances, fishing has evolved to suit the cosmos with various features, like auto-reeling."